config/proto/targets.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto2";
+
+package hermes.targets
+
+option go_package = "github.com/googleinterns/hermes/targets/proto";
+
+message TargetsDef {
```

Comment:
Add comment at the top describing what the intent of the message

Reply:
Okay:
```suggestion
// Definition of a Target, a storage system to be probed by a specific probe.
message TargetsDef {
```